<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>网络连接</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP地址</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_IP">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>端口</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Port">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Connection">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>lineEdit_IP</zorder>
    <zorder>label_2</zorder>
    <zorder>lineEdit_Port</zorder>
    <zorder>pushButton_Connection</zorder>
    <zorder>pushButton_Connection</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>481</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>数据显示</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>461</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>接收服务端数据</string>
     </property>
     <widget class="QTextEdit" name="textEdit_Recv">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>421</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>391</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>客户端数据推送</string>
     </property>
     <widget class="QTextEdit" name="textEdit_Send">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Send">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
